build/params_mainnet.go

```diff
@@ -70,7 +72,20 @@ const UpgradeChocolateHeight = 1231620
 // 2022-03-01T15:00:00Z
 var UpgradeOhSnapHeight = abi.ChainEpoch(1594680)
 
+var SupportedProofTypes = []abi.RegisteredSealProof{
+	abi.RegisteredSealProof_StackedDrg32GiBV1,
+	abi.RegisteredSealProof_StackedDrg64GiBV1,
+}
+var ConsensusMinerMinPower = abi.NewStoragePower(10 << 40)
+var MinVerifiedDealSize = abi.NewStoragePower(1 << 20)
+var PreCommitChallengeDelay = abi.ChainEpoch(150)
+
 func init() {
+	policy.SetSupportedProofTypes(SupportedProofTypes...)
```

Comment:
Let's not explicitly set these -- i think it's better to leave them at their defaults.

Reply:
(The variables above can totally stay, though)